exercises/raindrops/test/test_raindrops.c

```diff
@@ -1,5 +1,6 @@
 #include "vendor/unity.h"
 #include "../src/raindrops.h"
+#include <stdlib.h>
 
 #define BUFFER_LENGTH 100
```

Comment:
Don't think this is used anymore? Value could be changed to `16` and used in array definition in `convert_drops()`